### ISSUE TITLE
Expected class type but received "type" in type() bases argument

### ISSUE THREAD

**Issue by Gobot1234** [microsoft/pyright#3825]
**Describe the bug**
Passing a tuple or dict (there might be other classes) as a base that's created through the tuple constructor type fails to type check. 

**To Reproduce**
```py
type(
    "", tuple({tuple}), {},
    #   ^^^^^^^^^^^^^ Expected class type but received "type"
)
type("", tuple({dict}), {},)  # not fine
type("", tuple({int}), {},)  # fine
type("", tuple({type}), {},)  # also fine
```

**VS Code extension or command-line**
pylance: v2022.8.20

**Comment by erictraut** [comment]
This will be addressed in the next release.

**Comment by erictraut** [comment]
This is fixed in pyright 1.1.267, which I just published. It will also be included in a future version of pylance.
